plugins/notifications-backend/src/service/router.ts

```diff
@@ -62,12 +76,24 @@ export async function createRouter(
       throw new Error('either pageSize or pageNumber is not a number');
     }
 
+    const orderByIndex = FieldsAllowedToOrderBy.findIndex(f => f === orderBy);
+    const direction = orderByDirec?.toString();
+    if (orderByIndex < 0 || !['asc', 'desc'].includes(direction)) {
+      throw new Error(
+        `The orderBy parameter can be one of ${FieldsAllowedToOrderBy.join(
+          ',',
+        )}. The orderByDirec can be either 'asc' or 'desc'.`,
+      );
+    }
+
     getNotifications(
       dbClient,
       catalogClient,
       request.query,
       pageSizeNum,
       pageNumberNum,
+      FieldsAllowedToOrderBy[orderByIndex],
```

Comment:
i think we hsould pass the object NotificationsQuerySorting. what if we want to support multiple sorting columns? this way the router does not change.